src/dogstatsd/test/client.cc

```diff
@@ -66,9 +66,10 @@ void _test_tags(const char *tags, const char *const_tags, const char *expect) {
   getnameinfo((sockaddr *)&server.addr, server.addr.sin_len, host, NI_MAXHOST,
               port, NI_MAXSERV, NI_NUMERICHOST | NI_NUMERICSERV);
 
-  dogstatsd_client client =
-      dogstatsd_client_ctor(host, port, buf, len, const_tags);
-  REQUIRE(!dogstatsd_client_is_default_client(&client));
+  struct addrinfo *addrs = nullptr;
```

Comment:
It looks like `nullptr` was [added in GCC 4.6](https://gcc.gnu.org/projects/cxx-status.html). Do all the supported OS's have at least this version? We upgrade [GCC in our CentOS containers](https://github.com/DataDog/docker-library/blob/master/dd-trace-php/Dockerfile_centos#L11-L13), but do we know if the officially supported GCC version in CentOS 6 is 4.6 or greater? Or is this even an issue at all since it's just in the tests?

Reply:
I use a few C++ 11 features, like closures. I think all our operating systems support C++11 except native CentOS 6/7, and we use software collections there so no worries.

Reply:
This does remind me that as our client gets more advanced we probably do need to start running its standalone tests on every OS.